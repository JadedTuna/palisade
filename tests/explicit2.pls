low arr[3] = [10, 3, 6];
high secretNum = 99;
arr[2] = secretNum;