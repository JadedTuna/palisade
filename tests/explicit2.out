[1;35msecurity error: [0minsecure explicit flow
   1 | low arr[3] = [10, 3, 6];
   2 | high secretNum = 99;
   3 | arr[2] [1;35m=[0m secretNum;
       [1;35m~~~~~~~[0m[1;35m^[0m
